### DISCUSSION TITLE
Funnel : Plugin

### DISCUSSION THREAD

**Discussion by nachopv** [grafana/grafana#41667] [Plugins]
Hi 

Is there any plans of enabling  a Funnel plugin?

https://chartio.com/learn/charts/funnel-chart-complete-guide/

Basically we would be interested into plotting conversion ratios across an ecommerce purchase flow

**Comment by leeoniya** [comment]
no immediate plans to my knowledge. cc @grafana/grafana-bi-squad, @davkal 

the suggested alternatives on that page are pretty close though, which we already support: https://chartio.com/learn/charts/funnel-chart-complete-guide/#bar-chart.

![image](https://user-images.githubusercontent.com/43234/141701025-9b04e3b5-b0dd-4c10-993e-f121f1199836.png)

**Comment by ryantxu** [comment]
or maybe:
https://github.com/mckn/grafana-funnel-chart

**Comment by nachopv** [comment]
I do not see that one available in order to install it on Grafana Cloud ...
So i cannot say much on how mature it can be
I was thinking more likely on a corporate plugin approach rather than a personal initiative
This feature is a widely adopted visualization, and with relevant adoption in other tools such as Kibana and GA

We are currently workarounding it with bar charts but obviously is not simulating the capabilities of a native funnel , such as progres %, abandon rates , etc

**Comment by mckn** [comment]
I have started on a funnel chart and can put some more effort in to making it fully functioning if this is something you would like to use.

**Reply by samjewell** [reply]
@mckn Just wondered-  have you made any more progress on your funnel chart since your last message here? What are your plans for it?

**Reply by mckn** [reply]
I have not made any more progress I'm afraid. Having trouble finding time to move this forward but I will book a slot in my calendar every week where I can make some progress on this.

**Comment by mckn** [comment]
Here is an example where I use the echarts plugin to create a funnel chart:
https://gist.github.com/mckn/a0721af1f9aaa7cb103bd4752cae1443

The following plugins are required for the dashboard to function:
https://grafana.com/grafana/plugins/bilibala-echarts-panel/
https://grafana.com/grafana/plugins/marcusolsson-static-datasource/

<img width="787" alt="Skärmavbild 2023-01-12 kl  10 44 36" src="https://user-images.githubusercontent.com/172951/212033165-ef2c48fa-57d6-41ec-80dd-22718b812439.png">

**Reply by mckn** [reply]
Adding this for reference: https://grafana.com/grafana/plugins/mckn-funnel-panel/

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
